contrib/run_task.sh

```diff
@@ -69,12 +97,33 @@ main() {
 	schemars)
 	    do_schemars
 	    ;;
-	*)
-	    err "Error: unknown task $task"
-	    ;;
+
+        *)
+            err "Error: unknown task $task"
+            ;;
     esac
 }
 
+# Build and test for each crate, done with each toolchain.
+build_and_test() {
+    # Building the fuzz crate is more-or-less just a sanity check.
+    # pushd "$REPO_DIR/fuzz" > /dev/null
+    # cargo --locked build
```

Comment:
In 39f23d937289891e9e1bd89d14f804b2c517c3e0:

If we are going to comment these out we might as well just remove them.

Reply:
Ah that's a fail while testing the script yesterday. Big diffs are bad even for the developer :)